offchain/logs/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use clap::Parser;
+use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+
+#[derive(Clone, Parser, Debug)]
+#[command(name = "log_config")]
+#[command(about = "Configuration for Logs")]
+pub struct LogsEnvCliConfig {
+    #[arg(long, env, default_value = "false")]
+    pub logs_enable_timestamp: bool,
+
+    #[arg(long, env, default_value = "false")]
+    pub logs_enable_color: bool,
+}
+
+#[derive(Clone, Debug)]
+pub struct LogsConfig {
+    pub enable_timestamp: bool,
+    pub enable_color: bool,
+}
+
+impl LogsConfig {
+    pub fn initialize(env_cli_config: LogsEnvCliConfig) -> Self {
+        let enable_timestamp = env_cli_config.logs_enable_timestamp;
+
+        let enable_color = env_cli_config.logs_enable_color;
+
+        LogsConfig {
+            enable_timestamp,
+            enable_color,
+        }
+    }
+
+    pub fn default() -> Self {
+        LogsConfig {
+            enable_timestamp: false,
+            enable_color: false,
+        }
+    }
+}
+
+impl From<LogsEnvCliConfig> for LogsConfig {
+    fn from(cli_config: LogsEnvCliConfig) -> LogsConfig {
+        LogsConfig::initialize(cli_config)
+    }
+}
+
+pub fn initialize_logs(config: &LogsConfig) {
```

Comment:
```suggestion
pub fn configure(config: &LogsConfig) {
```
I'm suggesting the name change for two main reasons:

1) Losing the `logs` suffix reads better when calling the function with its fully classified name: `logs::initialize`
2) `initialize` reminds me of factory functions/methods. As we are not returning a new object from it, IMO `configure` seems a more appropriate verb.